archetypes/Pools/AddAltPoolModal/index.tsx

```diff
@@ -48,12 +52,18 @@ export default (({ open, onClose, sortedFilteredTokens }) => {
     };
 
     useEffect(() => {
-        if (isValidAddress) {
-            setImportMsg(pool.warning);
-        } else {
+        const isAvailable =
+            (poolLists?.TracerUnverified.pools.filter((v) => v.address === userInput) as StaticPoolInfo[]).length > 0; // Check if Pool exists in list of unverified
```

Comment:
I think the optional chain here could result in `undefined` and then we can accidentally try access `.length` on it, should ensure its an array and also convert casing in the boolean check to ensure a match

```suggestion
            ((poolLists?.TracerUnverified.pools || []).filter((v) => v.address === userInput) as StaticPoolInfo[]).length > 0; // Check if Pool exists in list of unverified
```